tests/torch/test_graph_building.py

```diff
@@ -86,40 +86,6 @@ def forward(self, x):
         unique_op_exec_contexts.add(node_op_address)
 
 
-def test_forward_trace_functor():
```

Comment:
We still have to test the forwarding - it's not the functor now, but the function is being tested. Keep the test, replace `func` with direct calls to `forward_trace_only`.